invenio_rdm_migrator/streams/communities/table_generator.py

```diff
@@ -45,6 +70,11 @@ def _generate_rows(self, data, **kwargs):
             member["community_id"] = community_id
             yield CommunityMember(**member)
 
+        featured_community = data["featured_community"]
+        if featured_community["id"]:
```

Comment:
Same as before. When a community is not featured, `featured_community` should be empty (or maybe no even exist).